docs/aerodromes/classc/Gingin.md

```diff
@@ -4,7 +4,36 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login ID              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| **Gingin ADC**    | **Gingin Tower**  | **118.500**    | **GIG_TWR**        |
+
+## Airsapce
+ ADC owns the Class C airspace **in the PE CTR** within 5nm of the YPEA ARP from `SFC` to `A035`. This airspace is primarily used for military circuits and initial and pitch approaches.
```

Comment:
Is ADC's airspace clipped by the 12TAC step like the circuit area is? Or does GIG ADC have ownership over that?

Reply:
ADC is clipped, its better explained with the diagram from the ad2 sup